spec/integration/shareable_task_spec.rb

```diff
@@ -82,6 +82,23 @@
     include_examples "invalid metadata"
   end
 
+  describe 'over docker', docker: true do
```

Comment:
we may want to move this to the transport shared examples

Reply:
I guess there's not much reason for it to use the CLI.

Reply:
Actually these tests cover several aspects of shareable tasks: that Puppet finds all the relevant files, and that we put them in the right place. We don't tend to use fixtures in the unit tests section. So I think this separation still makes sense.

Reply:
But then the cli_spec makes me a liar :)

Reply:
but we can test those aspects separately. Once with a single transport for integration and then at the transport level.

Reply:
Updated.